optuna/integration/tensorflow.py

```diff
@@ -43,25 +46,28 @@ class TensorFlowPruningHook(SessionRunHook):
             An estimator which you will use.
         metric:
             An evaluation metric for pruning, e.g., ``accuracy`` and ``loss``.
-        is_higher_better:
-           It should be :obj:`True` if you use a metric to be maximize such as ``accuracy``.
         run_every_steps:
            An interval to watch the summary file.
+        is_higher_better:
+           Please do not use this argument because this class refers to
+           :class:`~optuna.structs.StudyDirection` to check whether the current study is
+           ``minimize`` or ``maximize``.
     """
 
-    # TODO(sano): Remove is_higher_better after implementing maximize.
-    # TODO(sano): Get the information from StudyDirection.
-    def __init__(self, trial, estimator, metric, is_higher_better, run_every_steps):
-        # type: (optuna.trial.Trial, tf.estimator.Estimator, str, bool, int) -> None
+    def __init__(self, trial, estimator, metric, run_every_steps, is_higher_better=None):
+        # type: (optuna.trial.Trial, tf.estimator.Estimator, str, int, Optional[bool]) -> None
 
         self.trial = trial
         self.estimator = estimator
         self.current_summary_step = -1
         self.metric = metric
-        self.is_higher_better = is_higher_better
         self.global_step_tensor = None
         self.timer = tf.train.SecondOrStepTimer(every_secs=None, every_steps=run_every_steps)
 
+        if is_higher_better is not None:
+            raise ValueError('Please do not use is_higher_better argument of'
```

Comment:
We might want to clarify the reason, instead of just saying "please do not use...". E.g., `is_higher_better argument is obsolete since Optuna 0.9.0.`

Reply:
Thank you for your suggestion. I added the sentence you proposed.